client/console/src/main/java/org/apache/syncope/client/console/panels/MergeLinkedAccountsResourcesPanel.java

```diff
@@ -0,0 +1,174 @@
+import org.apache.syncope.client.console.commons.DirectoryDataProvider;
+import org.apache.syncope.client.console.commons.MergeLinkedAccountsWizardModel;
+import org.apache.syncope.client.console.commons.SortableDataProviderComparator;
+import org.apache.syncope.client.console.pages.BasePage;
+import org.apache.syncope.client.console.rest.ResourceRestClient;
+import org.apache.syncope.client.console.wicket.markup.html.form.ActionLink;
+import org.apache.syncope.client.console.wicket.markup.html.form.ActionsPanel;
+import org.apache.syncope.common.lib.to.ResourceTO;
+import org.apache.syncope.common.lib.types.StandardEntitlement;
+import org.apache.wicket.PageReference;
+import org.apache.wicket.ajax.AjaxRequestTarget;
+import org.apache.wicket.extensions.markup.html.repeater.data.sort.SortOrder;
+import org.apache.wicket.extensions.markup.html.repeater.data.table.IColumn;
+import org.apache.wicket.extensions.markup.html.repeater.data.table.PropertyColumn;
+import org.apache.wicket.extensions.wizard.WizardModel.ICondition;
+import org.apache.wicket.extensions.wizard.WizardStep;
+import org.apache.wicket.model.CompoundPropertyModel;
+import org.apache.wicket.model.IModel;
+import org.apache.wicket.model.Model;
+import org.apache.wicket.model.ResourceModel;
+import org.apache.wicket.model.StringResourceModel;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+
+public class MergeLinkedAccountsResourcesPanel extends WizardStep implements ICondition {
+    private static final long serialVersionUID = 1221037007528732347L;
+
+    private final MergeLinkedAccountsWizardModel wizardModel;
+
+    public MergeLinkedAccountsResourcesPanel(final MergeLinkedAccountsWizardModel wizardModel,
+                                             final PageReference pageReference) {
+        super();
+        setOutputMarkupId(true);
+        this.wizardModel = wizardModel;
+        add(new ResourceSelectionDirectoryPanel("resources", pageReference));
+    }
+
+    @Override
+    public boolean evaluate() {
+        return SyncopeConsoleApplication.get().getSecuritySettings().getAuthorizationStrategy().
+            isActionAuthorized(this, RENDER);
+    }
+
+    @Override
+    public String getTitle() {
+        setSummaryModel(new StringResourceModel("mergeLinkedAccounts.searchResource.summary",
+            Model.of(wizardModel.getMergingUser())));
+        setTitleModel(new StringResourceModel("mergeLinkedAccounts.searchResource.title",
+            Model.of(wizardModel.getMergingUser())));
+        return super.getTitle();
+    }
+
+    private class ResourceSelectionDirectoryPanel extends
+        DirectoryPanel<ResourceTO, ResourceTO,
+            ResourceSelectionDirectoryPanel.ResourcesDataProvider, ResourceRestClient> {
+
+        private static final String PAGINATOR_ROWS = "linked.account.merge.paginator.rows";
```

Comment:
You could actually drop this and reuse `Constants.PREF_RESOURCES_PAGINATOR_ROWS`